src/uwtools/tests/config/formats/test_ini.py

```diff
@@ -1,44 +1,25 @@
+# pylint: disable=duplicate-code
 """
 Tests for uwtools.config.formats.ini module.
 """
 
 import filecmp
-from typing import Any, Dict
 
 from uwtools.config.formats.ini import INIConfig
 from uwtools.tests.support import fixture_path
 
 # Tests
 
 
-def test_bash(salad_base, tmp_path):
-    """
-    Test that INI config load and dump work with a basic bash file.
-    """
-    infile = fixture_path("simple.sh")
-    outfile = tmp_path / "outfile.sh"
-    cfgobj = INIConfig(infile, space_around_delimiters=False)
-    expected: Dict[str, Any] = {
-        **salad_base["salad"],
-        "how_many": "12",
-    }  # str "12" (not int 12) for INI
-    assert cfgobj == expected
-    cfgobj.dump(outfile)
-    assert filecmp.cmp(infile, outfile)
-    cfgobj.update({"dressing": ["ranch", "italian"]})
-    expected["dressing"] = ["ranch", "italian"]
-    assert cfgobj == expected
-
-
 def test_parse_include():
     """
     Test that non-YAML handles include tags properly for INI with no sections.
```

Comment:
I know that it's not new, but what does "non-YAML" add to this docstring (and in the sh tests below)? And `no sections` is inaccurate, right? Could this be simplified? (`no sections` is accurate for the sh tests, I think.)